source/wp-content/themes/wporg-showcase-2022/src/site-breadcrumbs/style.scss

```diff
@@ -1,7 +1,7 @@
 .wp-block-wporg-site-breadcrumbs > div {
 	display: flex;
 	align-items: center;
-	min-height: var(--local-header-height);
+	font-size: var(--wp--preset--font-size--small);
```

Comment:
Rather than set this here, this setting is available on the block itself so it should be changed in `patterns/subnav-bar.php`:

```diff
- <!-- wp:wporg/site-breadcrumbs {"textColor":"charcoal-1"} /-->
+ <!-- wp:wporg/site-breadcrumbs {"textColor":"charcoal-1","fontSize":"small"} /-->
```

Reply:
Ah, thanks! 👍